questionpy_server/factories/__init__.py

```diff
@@ -1,10 +1,10 @@
 from .package import PackageInfoFactory
 from .attempt import AttemptFactory, AttemptScoredFactory, AttemptStartedFactory
-from .question_state import QuestionStateHash
+from .question_state import RequestBaseData
 
 
 __all__ = [
     'PackageInfoFactory',
     'AttemptFactory', 'AttemptScoredFactory', 'AttemptStartedFactory',
-    'QuestionStateHash'
+    'RequestBaseData'
```

Comment:
Hier sollte die Factory statt das Model stehen. (Der Import muss dann auch noch angepasst werden.)

```suggestion
    'RequestBaseDataFactory'
```